### PR TITLE
Feature/ID-369/CRUD functions for user account data management

### PR DESCRIPTION
**Pull request by JeferssonCL**
## Author
@JeferssonCL 
@SamuelEscalera 

## What?
- Implementation of the b-tree base to use in the project
- Package architecture definition for files

### CODE REVIEW

**Review by fabianromeroclaros — APPROVED** [review]

File: app/src/main/java/com/isc/hermes/database/AccountInfoManager.java [hunk 1]
```diff
@@ -35,9 +42,79 @@ public AccountInfoManager() {
...
+    public User getUserById(String userId) throws ExecutionException, InterruptedException, JSONException {
+        String apiUrl = ACCOUNT_INFO_COLLECTION + "/" + userId;
+        Future<String> responseData = apiHandler.getFutureCollectionString(apiUrl);
+        JSONObject jsonObject = new JSONObject(responseData.get());
+        return new User(jsonObject.getString("email"), jsonObject.getString("fullName"),
+                jsonObject.getString("userName"), jsonObject.getString("typeUser"),
+                jsonObject.getString("_id"), jsonObject.getString("pathImageUser"));
+    }
+
+    public boolean verifyIfAccountIsRegistered(String email) throws ExecutionException, InterruptedException, JSONException {
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/database/AccountInfoManager.java:78]: this method needs javadocs

File: app/src/main/java/com/isc/hermes/database/AccountInfoManager.java [hunk 2]
```diff
@@ -35,9 +42,79 @@ public AccountInfoManager() {
...
+        boolean isRegistered = false;
+        JSONArray arrayAccounts = getJSONArrayByEmail(email);
+        if (arrayAccounts.length() != 0) {
+            isRegistered = true;
+        } return isRegistered;
+    }
+
+    private JSONArray getJSONArrayByEmail(String email) throws ExecutionException, InterruptedException, JSONException {
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/database/AccountInfoManager.java:95]: This method needs javadocs

File: app/src/main/java/com/isc/hermes/database/AccountInfoManager.java (hunk 1 above)
Comment (SamuelEscalera) [reply app/src/main/java/com/isc/hermes/database/AccountInfoManager.java:78]: Corrected

File: app/src/main/java/com/isc/hermes/database/AccountInfoManager.java (hunk 2 above)
Comment (SamuelEscalera) [reply app/src/main/java/com/isc/hermes/database/AccountInfoManager.java:95]: Corrected

**Review by VillalobosGarciaGaby — APPROVED** [review]

**Review by Elvis-Castro-H — APPROVED** [review]
Approved

**Pull request merged**